src/components/Sidebar/index.module.css

```diff
@@ -0,0 +1,44 @@
+.sidebar {
+  position: fixed;
+  top: 0;
+  left: -260px;
+  width: 400px;
+  height: 100%;
+  padding: 12rem 1rem 6rem 7rem;
+  background-color: var(--blackHole);
+  z-index: 5;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  transition: left 0.5s ease;
```

Comment:
I would suggest use transform: translateX instead of left. 

It is better for perfomance

https://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/